specification/storagetasks/resource-manager/Microsoft.StorageTasks/stable/2023-01-01/storagetasks.json

```diff
@@ -0,0 +1,903 @@
+            "ValidateSubscriptionQuotaEnd",
+            "Creating",
+            "Succeeded",
+            "Deleting",
+            "Canceled",
+            "Failed"
+          ],
+          "x-ms-enum": {
+            "name": "ProvisioningState",
+            "modelAsString": false
+          }
+        },
+        "creationTimeInUtc": {
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "description": "The creation date and time of the storage task in UTC."
+        }
+      },
+      "required": [
+        "enabled",
+        "description",
+        "action"
+      ],
+      "description": "Properties of the storage task."
+    },
+    "StorageTaskAction": {
+      "type": "object",
+      "properties": {
+        "if": {
+          "$ref": "#/definitions/IfCondition",
+          "description": "The if block of storage task operation"
+        },
+        "else": {
+          "$ref": "#/definitions/ElseCondition",
+          "description": "The else block of storage task operation"
+        }
+      },
+      "required": [
+        "if"
+      ],
+      "description": "The storage task action represents conditional statements and operations to be performed on target objects."
+    },
+    "IfCondition": {
+      "type": "object",
+      "properties": {
+        "condition": {
+          "type": "string",
+          "description": "The condition predicate which is composed of object properties, eg: blob and container properties."
+        },
+        "operations": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/StorageTaskOperation"
+          },
+          "x-ms-identifiers": [],
+          "description": "List of operations to execute when the condition predicate satisfies."
+        }
+      },
+      "required": [
+        "condition",
+        "operations"
+      ],
+      "description": "The if block of storage task operation"
+    },
+    "ElseCondition": {
+      "type": "object",
+      "properties": {
+        "operations": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/StorageTaskOperation"
+          },
+          "x-ms-identifiers": [],
+          "description": "List of operations to execute in the else block"
+        }
+      },
+      "required": [
+        "operations"
+      ],
+      "description": "The else block of storage task operation"
+    },
+    "StorageTaskOperation": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The operation to be performed on the object.",
+          "enum": [
+            "SetBlobTier",
+            "SetBlobTags",
+            "SetBlobImmutabilityPolicy",
+            "SetBlobLegalHold",
+            "SetBlobExpiry",
+            "DeleteBlob",
+            "UndeleteBlob"
+          ],
+          "x-ms-enum": {
+            "name": "StorageTaskOperation",
```

Comment:
"StorageTaskOperation" looks duplicated with the definition "StorageTaskOperation" on line 660.

Reply:
Changed to StorageTaskOperationName